src/app/visual/vertical-tabset/vertical-tabs-visual.component.html

```diff
@@ -5,7 +5,7 @@
 
 <h2>Vertical tabset</h2>
 <div id="screenshot-vertical-tabset" [@.disabled]="true">
-  <sky-vertical-tabset showTabsText='Tab list'>
+  <sky-vertical-tabset [maintainTabContent]="maintainTabContent" showTabsText='Tab list'>
```

Comment:
Nitpicky again, but could we break this onto multiple lines and have the `maintainTabContent` be second?

Reply:
👍